src/components/Sidebar/sidebar.scss

```
@@ -62,16 +62,19 @@
   display: flex;
   align-items: center;
 }
-.dapp-nav a {
-  margin-top: 0.7rem;
-  margin-left: 33px;
-  align-items: center;
-  margin-bottom: 0.5rem;
-  svg {
-    width: 20px;
-    height: 20px;
-    margin-right: 12px;
-    vertical-align: -4px;
+.dapp-nav {
+  a,
+  .swap-title {
+    margin-top: 0.7rem;
+    margin-left: 33px;
+    align-items: center;
+    margin-bottom: 0.5rem;
```

Using shorthand for code elimination and best practices 🙌 

```suggestion
    margin: 0.7rem 0 0.5rem 33px;
    align-items: center;
```

What is our base font-size? Is it 10px? 
Anyway, there should be the same unit for consistency 
`margin-left: 3.3rem`?